fibo/conftest.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import requests
+from dev_classes.phase4_UI.jira_pages import LoginPO
+from dev_classes.phase4_UI.projectWd import ProjectWD
+from dev_classes.phase2_API.json_fixtures import *
+from dev_classes.phase2_API.properties_for_phase2 import *
+from dev_classes.phase4_UI.properties4 import user_name, user_pass, pytestScope
+
+
+# if (pytestScope='function')  -> Webdriver will init/teardown per... suddenly function!
+# if (pytestScope='class')  -> Webdriver will init/teardown per class
+@pytest.fixture(scope=pytestScope)
+def wd():
```

Comment:
The name of this fixture has similar name with wd variable.
Could you update the method name for better code readability?